src/lando/utils/management/commands/setup_dev.py

```diff
@@ -14,33 +16,44 @@ class Command(BaseCommand):
     def setup_workers(self):
         """Ensure a git and an hg worker exist on the local environment."""
         # Set up two workers, one for each SCM.
-        workers = {
-            SCM_TYPE_GIT: None,
-            SCM_TYPE_HG: None,
-        }
+        scm_types = [
+            SCM_TYPE_GIT,
+            SCM_TYPE_HG,
+        ]
+
+        worker_types = [
+            "",  # landing workers
+            "-automation-worker",
+        ]
```

Comment:
Nit: I'm going to remove the hg automation worker shortly as it is unused. Perhaps we could have an explicit list of the expected workers, `hg`, `git` and `git-automation-worker`, to be more precise.

Reply:
Ok, I added a more flexible mapping :+1:.